torch/testing/_internal/common_methods_invocations.py

```diff
@@ -629,6 +629,7 @@ def __init__(self,
                                                    # defaults to the value of `supports_forward_ad`
                  check_inplace_batched_forward_grad=None,   # whether to check batched forward grad when doing gradcheck
                                                             # defaults to the value of `check_batched_forward_grad`
+                 check_decomp=False,  # whether to check against torch._decomp
```

Comment:
My main goal is I want to be able to run tests for all decompositions quickly; I don't want to waste time churning through forward-backwards for everything in the OpInfo, when only twenty of the operators actually do anything nontrivial.

The test cannot easily query if a decomposition exists for an operator because OpInfos deal with torch API, and decompositions are registered against ATen API. I could dump a table mapping torch functions to aten functions (this would be relatively easy to do), but you now have the same problem of keeping the table up to date (though, I suppose, one could have a separate test testing that the table is up to date). WDYT?

Reply:
Yeah that all sounds reasonable. A table is interesting but I wouldn't want to make it a requirement today but more like an option later. But don't all the OpInfos specify their ATen names anyway? Are those not usable?

Reply:
Ah, in this case it's because I need to know what the ATen names of the backwards are, but I suppose we can add that instead!